frontendActions/src/main/java/com/virtuslab/gitmachete/frontend/actions/dialogs/SlideInDialog.kt

```diff
@@ -23,12 +24,13 @@ data class SlideInOptions(
 class SlideInDialog
     constructor(
         project: Project, private val branchLayout: IBranchLayout, private val parentName: String
-    ) : DialogWrapper(project, true) {
+    ) : DialogWrapper(project, /* canBeParent */ true) {
 
   // this field is only ever meant to be written on UI thread
   private var branchName = ""
   private var reattach = false
   private var reattachCheckbox: JCheckBox? = null
+  private val rootNames = branchLayout.rootEntries.map { it.name }
```

Comment:
Any shortcut like `branchLayout.rootEntries*.name`? Or maybe it's Groovy-only

Reply:
I think think this one is the simpliest